cmake/f3dConfig.cmake.in

```diff
@@ -1,24 +1,47 @@
-# When this file is run by CMake through the find_package command, the
-# following targets will exist:
+# When this file is run by CMake through the find_package command
+#
+# The following variables will be defined:
+#
+#   f3d_BINDINGS_JAVA              Will be enabled if F3D was built with java bindings
+#   f3d_BINDINGS_PYTHON            Will be enabled if F3D was built with python bindings
+#   f3d_BUILD_APPLICATION          Will be enabled if F3D application was built
+#   f3d_CONFIG_DIR                 Path to F3D configuration directory, can be absolute or relative
+#   f3d_MODULE_EXR                 Will be enabled if F3D was built with OpenEXR images support
+#   f3d_MODULE_EXTERNAL_RENDERING  Will be enabled if F3D was built with external rendering support
+#   f3d_MODULE_RAYTRACING          Will be enabled if F3D was built with raytracing support
+#   f3d_PLUGINS_INSTALL_DIR        Path to the location to install plugins so that F3D can find them
+#
+# It is also possible to look for optional components.
+#
+# Optional components are:
+#
+# `application`: If found, it the following target will exist
 #
 #   f3d::f3d           Target that contains the F3D executable
 #
+# `library`: If found, it the following target will exist
+#
 #   f3d::libf3d        Target that contains the libf3d
 #
-# The following local variables will also be defined:
+# And the following local variable will also be defined
 #
-#   f3d_BUILD_APPLICATION          Will be enabled if F3D application was built
-#   f3d_MODULE_EXTERNAL_RENDERING  Will be enabled if F3D was built with external rendering support
-#   f3d_MODULE_RAYTRACING          Will be enabled if F3D was built with raytracing support
-#   f3d_MODULE_EXR                 Will be enabled if F3D was built with OpenEXR images support
-#   f3d_BINDINGS_PYTHON            Will be enabled if F3D was built with python bindings
-#   f3d_BINDINGS_JAVA              Will be enabled if F3D was built with java bindings
 #   f3d_INCLUDE_DIR                Absolute path to the include directory of public f3d library classes
-#   f3d_CONFIG_DIR                 Path to F3D configuration directory, can be absolute or relative
-#   f3d_PLUGINS_INSTALL_DIR        Relative path to F3D plugins install directory
+#
+# `plugin`: If found many plugin related cmake macros will be available as well as the following target:
```

Comment:
Should the component be named `pluginsdk`? `plugin` sounds like you requesting a plugin.

Reply:
i can change it